healthcheck/templateHealthcheck.go

```diff
@@ -14,14 +14,15 @@ CREATE TABLE IF NOT EXISTS {{metadata.schema}}.{{metadata.table}}
   query text,
   executed text,
   expected text,
+	operation text,
```

Comment:
Extra indentation?

Reply:
Invisible characters are the bane of my existence.